ingress-controllers/readme.adoc

```diff
@@ -210,4 +210,325 @@ of the Ingress resource:
     $ kubectl delete -f templates/alb-ingress-controller.yaml
     $ kubectl delete -f https://raw.githubusercontent.com/coreos/alb-ingress-controller/master/examples/default-backend.yaml
 
-Check in the EC2 console to ensure your ALB has been deleted.
\ No newline at end of file
+Check in the EC2 console to ensure your ALB has been deleted.
+
+== Kube AWS Ingress Controller and Skipper
+
+link:https://github.com/zalando-incubator/kubernetes-on-aws[Kube AWS Ingress Controller]
+creates AWS Application Load Balancer (ALB) that is used to terminate TLS connections and use
+link:https://aws.amazon.com/certificate-manager/[AWS Certificate Manager (ACM)] or
+link:http://docs.aws.amazon.com/IAM/latest/APIReference/Welcome.html[AWS Identity and Access Management (IAM)]
+certificates. ALBs are used to route traffic to an Ingress http router for example
+link:https://github.com/zalando/skipper/[skipper], which routes
```

Comment:
Highlight key value of skipper?

Reply:
It's written "green-blue deployments, feature toggles, reate limits, circuitbreakers, shadow traffic or A/B tests."